orderbook/openapi.yml

```diff
@@ -153,8 +153,34 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/FeeInformation"
-        400:
+        404:
           description: sellToken not existent
+  /api/v1/trades/:
+    get:
+      summary: Get existing Trades.
+      description: |
+        By default all trades are returned.
+        Queries can be refined by specifiying owner or order_uid.
+      parameters:
+        - name: owner
+          in: query
+          schema:
+            $ref: "#/components/schemas/Address"
+          required: false
+        - name: order_uid
```

Comment:
why snake case? also I wouldn't say `Uid` --> should it be `orderId`

Reply:
good catch, not snake_case but also not `OrderId` but rather `orderUid`